publisher/publisher.go

```diff
@@ -18,6 +18,7 @@ import (
 const (
 	AWSElasticLoadBalancerFormat = "aws_elb"
 	AWSCloudFrontWebFormat       = "aws_cf_web"
+	AWSCloudTrailFormat          = "aws_ct"
```

Comment:
Is this used anywhere? If we don't have any `aws_ct` entry in the nginx log format file, we can probably skip this.